app/src/main/java/com/example/lapnguyen/weather_3/screen/mainscreen/MainPresenter.java

```diff
@@ -0,0 +1,55 @@
+package com.example.lapnguyen.weather_3.screen.mainscreen;
+
+import android.icu.text.SimpleDateFormat;
```

Comment:
ERROR: Unused import: android.icu.text.SimpleDateFormat. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports